interview-questions.md

```diff
@@ -7,60 +7,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: An instance variable is when youre creating an instance inside the varaible. Calling on the instances or instantiating is another term when you are calling upon an action. 
 
-  Researched answer:
+  Researched answer: In object-oriented programming with classes, an instance variable is a variable defined in a class, for which each instantiated object of the class has a separate copy, or instance.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: I remember the blocks challenge while i was driving and not navigating that you needed a d0 in the process. 
 
-  Researched answer:
+  Researched answer:Ruby has anonymous functions similar to JavaScript. They are called blocks. Blocks can be created with the do/end keywords OR with {}. Blocks are code that is passed to a method.
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: From the note-taking I did under this topic I remember writing over and over that its when the return is automated once you return something in ruby without typing return.
```

Comment:
Providing an example from memory here will be really helpful for understanding. Good stuff though.